cmd/devp2p/discv4cmd.go

```diff
@@ -181,7 +181,7 @@ func discv4Crawl(ctx *cli.Context) error {
 }
 
 func parseBootnodes(ctx *cli.Context) ([]*enode.Node, error) {
-	s := params.RinkebyBootnodes
+	s := params.AlfajoresBootnodes
```

Comment:
Is there a specific reason why you are using the Alfajores bootnodes here?

Reply:
Looks like they just had some default here